test/functional/legacy/delete_spec.lua

```diff
@@ -52,6 +52,10 @@ describe('Test for delete()', function()
         silent !ln -s Xfile Xlink
       endif
     ]])
+    if eval('v:shell_error') == 1 then
```

Comment:
I've read that exit codes are not reliable on Windows, so maybe `eval('v:shell_error') != 0` is better practice.